tests/spec/fs.appendFile.spec.js

```diff
@@ -124,4 +124,20 @@ describe('fs.appendFile', function() {
       });
     });
   });
+  
+  it('should accept numbers and appends them to the file', function(done) {
+    var fs = util.fs();
+    var contents = 'This is a file.';
+    var more = 10000;
+
+    fs.appendFile('/myfile', more, 'utf8', function(error) {
+      if(error) throw error;
+
+      fs.readFile('/myfile', 'utf8', function(error, data) {
+        expect(error).not.to.exist;
+        expect(data).to.equal(contents + more);
```

Comment:
This doesn't make sense: you are expecting to have `'This is a file.10000'`, but you never write `contents` to disk.  You'd either need to add a `writeFile` above that does this first (before your `appendFile`, or just get rid of `contents` completely from this test and test for the contents to be `'10000'`.